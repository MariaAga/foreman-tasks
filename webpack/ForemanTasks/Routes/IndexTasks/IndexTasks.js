import React from 'react';
import PropTypes from 'prop-types';
import TasksTablePage from '../../Components/TasksTable';

const IndexTasks = ({ history }) => (
  <div>
    <TasksTablePage history={history} />
  </div>
);
IndexTasks.propTypes = {
  history: PropTypes.object.isRequired,
};

export default IndexTasks;
