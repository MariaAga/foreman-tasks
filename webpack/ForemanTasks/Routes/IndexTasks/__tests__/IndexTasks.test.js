import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';

import IndexTasks from '../IndexTasks';

const fixtures = {
  'render without Props': {
    history: {
      push: jest.fn(),
    },
  },
};

describe('IndexTasks', () =>
  testComponentSnapshotsWithFixtures(IndexTasks, fixtures));
