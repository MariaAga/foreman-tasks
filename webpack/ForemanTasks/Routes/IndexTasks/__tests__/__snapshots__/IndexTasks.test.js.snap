// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexTasks render without Props 1`] = `
<div>
  <Connect(TasksTablePage)
    history={
      Object {
        "push": [MockFunction],
      }
    }
  />
</div>
`;
